module/Photo/view/photo/photo-admin/index.phtml

```diff
@@ -70,23 +70,23 @@ $this->headLink()
                     </tr>
                     <tr>
                         <td><?= $this->translate('Focal length') ?></td>
-                        <td><?= $photo->getFocalLength() ?></td>
+                        <td><?= is_null($photo->getFocalLength()) ? $this->translate("Unknown") : $photo->getFocalLength() ?></td>
                     </tr>
                     <tr>
                         <td><?= $this->translate('Exposure time') ?></td>
-                        <td><?= $photo->getExposureTime() ?></td>
+                        <td><?= is_null($photo->getExposureTime()) ? $this->translate("Unknown") : $photo->getExposureTime() ?></td>
                     </tr>
                     <tr>
                         <td><?= $this->translate('Shutter speed') ?></td>
-                        <td><?= $photo->getShutterSpeed() ?></td>
+                        <td><?= is_null($photo->getShutterSpeed()) ? $this->translate("Unknown") : $photo->getShutterSpeed() ?></td>
                     </tr>
                     <tr>
                         <td><?= $this->translate('Aperture') ?></td>
-                        <td><?= $photo->getAperture() ?></td>
+                        <td><?= is_null($photo->getAperture()) ? $this->translate("Unknown") : $photo->getAperture() ?></td>
                     </tr>
                     <tr>
                         <td><?= $this->translate('ISO') ?></td>
-                        <td><?= $photo->getIso() ?></td>
+                        <td><?= is_null($photo->getIso()) ? $this->translate("Unknown") : $photo->getIso() ?></td>
```

Comment:
Omdat die variabelen allen gegarandeerd getallen zijn. (is per database schema vastgelegd)